src/index.js

```diff
@@ -53,6 +54,9 @@ const defaultRuleProperties = {
   schemaJsonFilepath: {
     type: 'string',
   },
+  schemaString: {
```

Comment:
`makeExecutableSchema()` defines [`typeDefs`](http://dev.apollodata.com/tools/graphql-tools/generate-schema.html#makeExecutableSchema):

> typeDefs is a required argument and should be an array of GraphQL schema language strings or a function that takes no arguments and returns an array of GraphQL schema language strings. The order of the strings in the array is not important, but it must include a schema definition.

That's not exactly what's used here. Alternatively, [`buildSchema()`](http://graphql.org/graphql-js/utilities/#buildschema) just refers to it as `source`. [`printSchema()`](http://graphql.org/graphql-js/utilities/#printschema) refers to it as "schema in the Schema Language format".

I think `schemaString` is a decent compromise given the ambiguity and that the API is already using `schemaJson`.